docs/source/using-discovery/slurm.md

```diff
@@ -0,0 +1,769 @@
+
+(slurm-partitions)=
+### Partition(s)
+A 'partition' is a grouping of nodes. You can think of partitions as virtual clusters within your HPC system. They allow administrators to segregate the compute environment based on factors like job sizes, hardware type, or resource allocation policies.
+
+:::{seealso}
+Our {ref}`partition-names` documentation.
+:::
+
+### Account information
+Some Discovery users have more than one Discovery group account associated with their usernames. For example, a student might be in a class using Discovery and a student club using Discovery for a club project. In this case, the student would have two group accounts associated with their username.
+
+When running a job with either `srun` or `sbatch`, if you have more than one account associated with your username, we recommend you use the `--account=` flag and specify the account that corresponds to the respective project. In the example with a student associated with a class and a student club, if the student is on Discovery submitting a job for a project for their class, set the `account=` flag to the name of the class account. If the student is working on a project for the club, set the `account=` flag to the name of the student club account.
+
+To find out what account(s) your username is associated with, use the following command:
+
+:::{code} bash
+sacctmgr show associations user=<yourusername>
+:::
+
+After you have determined what accounts your username is associated with, if you have more than one account association, you can use the `account=` flag with your usual `srun` or `sbatch` commands.
+
+For example, if you are associated with an account named `dataclub` and an account named `info7500`, and you’re currently doing work that should be associated with the `dataclub` account, in your `srun` command, you can add the `--account=dataclub` flag to specify that account:
+
+:::{code} bash
+srun --account=dataclub --partition=short --nodes=1 --ntasks=28 --mem=0 --pty /bin/bash
+:::
+
+### Jobs, Job Steps, and Job Arrays
+A **job** in Slurm is a user-defined computational task that's submitted to the cluster for execution. Each job has one or more **job steps**, sub-tasks that are part of a larger job and can be executed in parallel or sequentially.
+
+**Job arrays** are a series of similar jobs that differ only by the array index. They're especially useful when you want to execute the same application with different inputs.
+
+### Tasks
+Tasks are the individual processes that run within a job step. They could be single-threaded or multi-threaded and can run on one or more nodes.
+
+### Understanding the Configuration File (slurm.conf)
+The slurm.conf file is the primary configuration file for Slurm. It contains the parameters that govern the behavior of the Slurm controller, nodes, and partitions.
```

Comment:
I would add that the typical user may never encounter a slurm.conf file. As written a user may think this is something they will need to modify to use slurm.

Reply:
I would put the slurm.conf info as possibly more info than needed (if wanting to cut to a more manageable page length).

Reply:
Okay. I added a note specifying this is just FYI and moved the code to a dropdown element (which is closed by default). This way, less room is taken, specification that it is not for the typical user, and the knowledge is still there for those interested in the scheduler. 

Does that look better?